run.py

```
@@ -0,0 +1,153 @@
+import argparse
+import csv
+from pathlib import Path
+import os
+import pickle
+import yaml
+
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer
+from bertopic.vectorizers import ClassTfidfTransformer
+from spacy.lang.en import stop_words
+from sentence_transformers import SentenceTransformer
+
+from PatientX.models.BERTopicModel import BERTopicModel
+from PatientX.MistralRepresentation import MistralRepresentation
+
+
+def read_data(datafolder: Path):
+    """
+    Read in data
+
+    datafolder: Path
+    Returns: List containing documents
+    """
+    dfs = []
+
+    assert(os.path.isdir(datafolder))
+
+    for filename in os.listdir(datafolder):
+        filepath = os.path.join(datafolder, filename)
+        if os.path.isfile(filepath):
+            try:
+                df = pd.read_csv(filepath)
+                dfs.append(df.copy(deep=True))
+            except csv.Error as e:
+                pass
```

Don't silently ignore the error.